components/motor/dimensionengineering/sabertooth.go

```diff
@@ -556,12 +549,13 @@ func goForMath(maxRPM, rpm, revolutions float64) (float64, time.Duration) {
 		rpm = -1 * maxRPM
 	}
 
-	if revolutions == 0 {
-		powerPct := rpm / maxRPM
-		return powerPct, 0
+	dir := 1.0
+	if rpm*revolutions == 0.0 {
+		dir = 0.0
```

Comment:
seems fine to me? but now I did add a check to gofor, which is before this point, so we shouldn't get here anyways, but I thought I should leave the 0 case in case goForMath is called directly somewhere else at a future point in time?